src/serialize.js

```diff
@@ -14,14 +16,9 @@ export default (tree, { search } = {}) =>
             if (!x.hasValue) x.markedForDeletion = true
           }
           _.each(unsetOn(_, x), [
-            'markedForUpdate',
-            'path',
-            'updating',
-            'paused',
-            'missedUpdate',
-            'hasValue',
-            'context',
+            ..._.keys(_.omit('lastUpdateTime', defaults)),
```

Comment:
you could omit lastUpdateTime conditionally on search instead of adding it conditionally below